axelrod/interaction_utils.py

```diff
@@ -239,36 +240,27 @@ def compute_sparklines(interactions, c_symbol='█', d_symbol=' '):
         sparkline(histories[1], c_symbol, d_symbol))
 
 
-def read_interactions_from_file(filename, progress_bar=True,
-                                num_interactions=False):
+def read_interactions_from_file(filename,
```

Comment:
one line?

Reply:
> Will review more -- github isn't loading the large diffs and I don't want to lose the comments I've already made.

Yeah, no rush with this at all. Small piece meal chunks is probably the way to go. Sorry the diff is so horrible.

> 
> Do we want to add an end user function to assist in reading the interactions file?

We've got `iu.read_interactions_from_file` which I've documented in 
843b008, I think that could be user facing (and in fact we could have another tutorial going over each function in `iu` ?) but let me know if you were thinking of something different.